hydpy/interfaces/rconcinterfaces.py

```diff
@@ -0,0 +1,31 @@
+"""This module defines submodel interfaces for calculating runoff
+concentration processes."""
+
+# import...
+# ...from hydpy
+from hydpy.core import modeltools
+from hydpy.core.typingtools import *
+from typing import Any
+
+class RConcModel_V1(modeltools.SubmodelInterface):
+    """Interface for calculating the unit hydrograph or linear storage cascade
+    output."""
+
+    typeid: ClassVar[Literal[1]] = 1
+    """Type identifier for |RConcModel_V1| submodels."""
+
+    @modeltools.abstractmodelmethod
+    def set_rconcinput(self, v: float) -> None:
+        """Set the runoff concentration input."""
+
+    @modeltools.abstractmodelmethod
+    def determine_rconcoutput(self) -> None:
+        """Calculate runoff concentration output."""
+
+    @modeltools.abstractmodelmethod
+    def get_rconcoutput(self) -> float:
+        """Get the previously calculated runoff concentration output."""
+
+    @modeltools.abstractmodelmethod
+    def get_waterbalance(self, initial_conditions: float) -> float:
```

Comment:
The annotation for `initial_conditions` is wrong.

Reply:
This was an open issue as written in personal message yesterday. Thanks for clarifying (solution is removing decorator as explained in next issue)